Mathlib/CategoryTheory/Endomorphism.lean

```diff
@@ -197,4 +197,88 @@ end FullyFaithful
 
 end Functor
 
+namespace Equivalence
+
+universe uC uC' vC vC' uD uD' vD vD'
+
+variable {C : Type uC} [Category.{vC} C]
+variable {C' : Type uC'} [Category.{vC'} C']
+variable {D : Type uD} [Category.{vD} D]
+variable {D' : Type uD'} [Category.{vD'} D']
+variable {f : C ⥤ D}  {g : C' ⥤ D'}
+variable {e : C ≌ C'} {e' : D ≌ D'}
+
+/--
+Suppose we have categories `C, C', D, D'` such that the diagram of functors
+```
+C ===== f =====> D
+||e            ||e'
+||             ||
+C' ==== g ====> D'
+```
+commutes where `e` and `e'` are equivalence of categories.
+
+Then we have an isomorphism of endomorphism monoids `End f ≃* End g'` and
+-/
+@[simps]
+def endMonoidEquiv
+    (sq₀ : f.comp e'.functor ≅ e.functor.comp g) : End f ≃* End g :=
```

Comment:
Apparently this can be a combination of [CategoryTheory.Functor.FullyFaithful.mulEquivEnd](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Endomorphism.html#CategoryTheory.Functor.FullyFaithful.mulEquivEnd), [CategoryTheory.Equivalence.congrLeft/Right](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Equivalence.html#CategoryTheory.Equivalence.congrRight) and [CategoryTheory.Iso.conj](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Conj.html#CategoryTheory.Iso.conj).